latte/utils/utils.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+from collections import defaultdict as dd
```

Comment:
I'm a big fan of using explicit names, i.e., `defaultdict` is easier to understand than `dd` (I needed to go back to the header to check it) :slightly_smiling_face: